### PR TITLE
Remove GitHub logo from header

### PR DESCRIPTION
**Pull request by pnax**
## Purpose

This removes the Github logo from the header. There still is a link in the footer.

## Context

Partially addresses #231.

## Changes

* Remove the Github logo and link from the navigation component.
* Update text link to source code in footer 
* Update footer rendering

## How to test this PR

The logo should not be displayed in the upper right corner on wide screen.

### CODE REVIEW

**Review by blacksponge — COMMENTED** [review]
I don't mind that the Github logo is removed from the header. It is indeed quite invasive. Maybe the text of the link in the footer could be changed to something more generic like "Source code" (or "Source code on Github" if we want to keep the Gihtub mention) or equivalent?

**Comment by matsduf** [comment]
If Github does not require that there is a link to and mentioning of Github I do think that we need to have that. I suggest "Source code and documentation" as the link text. Github as such is not interesting.

File: src/app/components/footer/footer.component.html [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-            <a class="text-dark" href="https://github.com/zonemaster/zonemaster">Github</a>
-          </div>
-        </div>
+      <div class="contact">
+        <i class="fa fa-envelope-o" aria-hidden="true"></i>
+        <a class="text-dark" href="mailto:{{contactAddress}}">{{contactAddress}}</a>
+      </div>
+      <div class="source-code">
+        <i class="fa fa-github" aria-hidden="true"></i>
+        <a class="text-dark" href="https://github.com/zonemaster/zonemaster" i18n>Source code</a>
```
Comment (matsduf) [inline src/app/components/footer/footer.component.html:13]: I think there is room for "and documentation" too.

**Comment by blacksponge** [comment]
Replaced by #403 and #411

**Pull request closed without merging**